Updated cluster create command to take in network.

ashcrow opened the pull request:
This will be used as part of the flanneld/etcd decoupling.

The update includes a new flag: `-n/--network`. It takes either
`flannel_etcd` or `flannel_service`. If not specified `flannel_etcd`
is used.

Example:

```
    $ commctl cluster create -n flannel_service
    Success
```

ashcrow committed:
Updated cluster create command to take in network.

The update includes a new flag: -n/--network. It represents the
network instance to use with the cluster. If no input is given
the network named default will be used.

Example:

    $ commctl cluster create -n network1 mycluster
    Success

ashcrow commented:
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;B5lCS4mu&#x2F;382-8-multi-host-mgr-decouple-flanneld-configuration-from-etcd"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> (8) [multi-host-mgr] Decouple flanneld configuration from etcd</a>

mbarnes commented:
LGTM.  Did you want this merged now or wait until the server piece is in place?

ashcrow commented:
@mbarnes let's wait. I may have a few more changes to make with it as I figure out the best way to define the network.

ashcrow commented:
:arrow_up:

mbarnes commented:
As with `commctl host` and `commctl cluster`, I think the network name should be a positional arg since it's mandatory (drop the `-n`/`--network`).  Aside from that, this all looks good.

ashcrow commented:
I'm going to amend the commit so that the commit message is correct with the above change.

mbarnes commented:
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;G93DmPOI&#x2F;407-refresh-atomic-man-page-for-new-commissaire-commands"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> Refresh atomic man page for new commissaire commands.</a>

mbarnes commented:
The `atomic` command will automatically pick this up so I added a card to update its man page.

(We should probably have our own man page too at some point.)

ashcrow committed:
Added new command: network.

Examples:

    $ commctl network create mynetwork
    Success
    $ commctl network create mynetwork -t flannel_server
    Success
    $ commctl network create mynetwork -t flannel_server -o {"address": "192.168.152.102:8080"}
    Success

ashcrow commented:
Updated. :arrow_up:

mbarnes commented:
Hmm... so :+1: to the `network` command change.

But I did prefer the `-n`/`--network` option for `cluster create` since that actually _is_ optional (has a default).  Sorry if I was unclear.  Or do you disagree?

ashcrow commented:
Ha no problem. I'll readd that back in this afternoon :laughing:

ashcrow commented:
:arrow_up:

mbarnes commented:
:+1: Sorry for the confusion.

@rh-atomic-bot r+
